database/RocksIterator.java

```diff
@@ -28,6 +28,7 @@
 import com.vaticle.typedb.core.graph.common.Storage.Key;
```

Comment:
RocksIterator & RocksStorage: statistics on the number and timings of DB interactions for internal use.